### PR TITLE
Bump Pangolin to 3.1.7

### PR DESCRIPTION
**Pull request by drpatelh**

### CODE REVIEW

**Review by jfy133 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -431,7 +433,7 @@ using a combination of `bwa` and `samtools` to output a BAM file instead of a SA
 
 - A module file SHOULD only define input and output files as command-line parameters to be executed within the process.
 
-- All other parameters MUST be provided as a string i.e. `options.args` where `options` is a Groovy Map that MUST be provided via the Nextflow `addParams` option when including the module via `include` in the parent workflow.
+- All `params` within the module MUST be initialised and used in the local context of the module. In other words, named `params` defined in the parent workflow MUST NOT be passed to the module to keep them as generic as possible and to allow developers to call their parameters whatever they want. In general, it may be more suitable to use additional `input` modules to cater for such scenarios.
```
Comment (JoseEspinosa) [inline README.md]: Do you mean "it may be more suitable to use additional `input` **value channels** to cater for such scenarios

Comment (drpatelh) [reply README.md]: Yep, sorry, I re-worded already but will add channels.

Comment (drpatelh) [reply README.md]: Done https://github.com/nf-core/modules/pull/593/commits/0f681b2beb52ae908c75278347ea67697e44e789

**Review by JoseEspinosa — APPROVED** [review]
LGTM :guitar:

**Pull request merged**